Formula/acme.rb

```diff
@@ -17,9 +16,9 @@ class Acme < Formula
   end
 
   devel do
-    url "https://web.archive.org/web/20150501011451/https://www.esw-heim.tu-clausthal.de/~marco/smorbrod/acme/current/acme093testing.tar.bz2"
-    sha256 "cf374869265981437181609483bdb6c43f7313f81cfe57357b0ac88578038c02"
-    version "0.93"
+    url "https://downloads.sourceforge.net/code-snapshots/svn/a/ac/acme-crossass/code-0/acme-crossass-code-0-98-trunk.zip"
+    sha256 "3e04526074e4b58b35f75de9ba84a4e9fdb25b252ed5b77ebad59dedd6f1430e"
+    version "0.98"
```

Comment:
Why would 0.98 be devel and 0.97 be stable? Where is the release numbering documented?

Reply:
The SVN revision numbers and the releases appear to be coincidental. Announcements are done on a case by case basis, e.g. [here](https://www.forum64.de/index.php?thread/77835-neues-acme-release/&postID=1216668#post1216668).

Reply:
The link does not show any 0.98 version. Let's remove the devel block, and keep a single version.